src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -633,7 +635,9 @@ export class ExerciseChatWidgetComponent implements OnInit, OnDestroy, AfterView
         if (error.status === 403) {
             this.stateStore.dispatch(new ConversationErrorOccurredAction(IrisErrorMessageKey.IRIS_DISABLED));
         } else if (error.status === 429) {
-            this.stateStore.dispatch(new ConversationErrorOccurredAction(IrisErrorMessageKey.RATE_LIMIT_EXCEEDED));
+            const map = new Map<string, any>();
```

Comment:
avoid the use of `any`

Reply:
Will be addressed in a separate PR, as this requires a bigger refactoring of the Iris client.